OpenOversight/app/main/forms.py

```diff
@@ -334,3 +334,17 @@ class IncidentForm(DateFieldForm):
     last_updated_id = HiddenField(validators=[Required(message='Incidents must have a user id for editing.')])
 
     submit = SubmitField(label='Submit')
+
+
+class BrowseForm(Form):
+    rank = SelectField('rank', default='Not Sure', choices=RANK_CHOICES,
+                       validators=[AnyOf(allowed_values(RANK_CHOICES))])
+    race = SelectField('race', default='Not Sure', choices=RACE_CHOICES,
+                       validators=[AnyOf(allowed_values(RACE_CHOICES))])
+    gender = SelectField('gender', default='Not Sure', choices=GENDER_CHOICES,
+                         validators=[AnyOf(allowed_values(GENDER_CHOICES))])
+    min_age = SelectField('min_age', default=16, choices=AGE_CHOICES,
```

Comment:
`'min_age' -> 'minimum age'` (this is the label that will be rendered in the webapp)